src/reusedBrowser.ts

```diff
@@ -225,6 +235,24 @@ export class ReusedBrowser implements vscodeTypes.Disposable {
     return this._recorderModeForTest;
   }
 
+  async connectMCP(mcp: McpConnection, model: TestModel) {
```

Comment:
I find this indirection undesirable. Let MCPConnection do all of this, and perhaps call `reusedBrowser.setKeepAlive(true/false)` to prevent auto-closure.